packages/esm-patient-chart-app/src/workspace/context-workspace.component.tsx

```diff
@@ -19,7 +19,8 @@ const ContextWorkspace: React.FC<RouteComponentProps<ContextWorkspaceParams>> =
   const { patientUuid } = match.params;
   const { active, title, closeWorkspace } = useWorkspace();
   const { t } = useTranslation();
-  const props = React.useMemo(() => ({ closeWorkspace, patientUuid }), [closeWorkspace, patientUuid]);
+  const light = layout === 'tablet';
+  const props = React.useMemo(() => ({ closeWorkspace, patientUuid, light }), [closeWorkspace, light, patientUuid]);
```

Comment:
I would rather return / call it `isTablet` or `layout`. `light` only makes sense given the context of a component - but has pretty much no (direct) meaning when inserted as props to a workspace.